android/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,10 @@
 
     <application>
         <activity android:name=".CrashActivity"></activity>
+
+        <meta-data
+                android:name="firebase_crashlytics_collection_enabled"
```

Comment:
what does this do ? Any equivalent for iOS ?

Reply:
It automatically initialises the whole Fabric if set to `true` (by default). We need to disable it to [disable the initialisation](https://firebase.google.com/docs/crashlytics/customize-crash-reports#android).

iOS also has something similar in the [Info.plist](https://firebase.google.com/docs/crashlytics/customize-crash-reports#ios)

Reply:
ho ok didn't know that :) thanks, I'll try to do the iOS part asap

Reply:
Thanks!

Reply:
In the same time if you can take a look at the issues I mention you ? I think there few problems with the stack parsing :/